include/libtorrent/add_torrent_params.hpp

```diff
@@ -151,7 +151,7 @@ namespace libtorrent
 			// in this ``add_torrent_params`` object will take precedence over
 			// whatever states are saved in the resume data. For instance, the
 			// ``paused``, ``auto_managed``, ``sequential_download``, ``seed_mode``,
-			// ``super_seeding``, ``max_uploads``, ``max_connections``,
+			// ``super_seeding``, ``priority``, ``max_uploads``, ``max_connections``,
```

Comment:
sorry, I was misleading in my first comment (because I misunderstood the context of your html edit). There's no priority field in add_torrent_params. please revert this file.